odoo/addons/base/models/res_lang.py

```diff
@@ -44,6 +47,12 @@ class LangDataDict(ReadonlyDict):
     """
     __slots__ = ()
 
+    def __getitem__(self, key: K) -> T:
```

Comment:
Keep the same typing than `__missing__`.
```suggestion
    def __getitem__(self, key: Any) -> LangData:
```